python/tests/catalog/test_base.py

```diff
@@ -78,14 +75,14 @@ def load_table(self, identifier: Union[str, Identifier]) -> Table:
         try:
             return self.__tables[identifier]
         except KeyError as error:
-            raise NoSuchTableError(f"Table does not exist: {identifier}") from error
+            raise NoSuchNamespaceError(f"Table does not exist: {identifier}") from error
 
     def drop_table(self, identifier: Union[str, Identifier]) -> None:
         identifier = Catalog.identifier_to_tuple(identifier)
         try:
             self.__tables.pop(identifier)
         except KeyError as error:
-            raise NoSuchTableError(f"Table does not exist: {identifier}") from error
+            raise NoSuchNamespaceError(f"Table does not exist: {identifier}") from error
```

Comment:
I pointed this out in the REST catalog, but this should still be `drop_table`. I think you were updating this based on my comment for the `list_tables` method. In that method, 404 means `NoSuchNamespaceError`, but for methods that identify a specific table, we want to use `NoSuchTableError`.

Reply:
Yes, I went over the list, sorry for the confusion